docker_util/__init__.py

```diff
@@ -0,0 +1,84 @@
+import json
+from pathlib import Path
+from subprocess import PIPE, run
+from typing import List
+
+DOCKER = 'docker'
+DOCKER_LIST_CONTAINER_COMMAND: List[str] = [
+    DOCKER,
+    'container',
+    'ls',
+    '--format',
+    '{{json .}}'
+]
+DOCKER_MIGRATE_COMMAND_TEMPLATE: List[str] = [
+    DOCKER,
+    'exec',
+    '{container_id}',
+]
+
+CONTAINER_LABEL_BASE = 'hubmap/data-portal-python'
+CONTAINER_SUFFIXES = [
+    'base',
+    'dev',
+    'prod',
+]
+# For local Docker override config
+CONTAINER_LABELS = {'dev-local_django'}
+CONTAINER_LABELS.update(f'{CONTAINER_LABEL_BASE}-{suffix}' for suffix in CONTAINER_SUFFIXES)
+
+REPOSITORY_PATH = Path('/opt/hubmap-data-portal')
+
+MANAGE_PY_COMMAND_TEMPLATE: List[str] = [
+    'python3',
+    str(REPOSITORY_PATH / 'hubmap/manage.py'),
+]
+
+def get_running_containers() -> List[str]:
```

Comment:
I'm not 100% sure what you mean by "one of my libraries", but this is all Python standard library and syntax. It's not enforced at runtime with standard Python interpreters; it's just for improved code analysis (e.g. _telling_ PyCharm the return type of a function instead of making the tool infer it).